lib/matplotlib/testing/widgets.py

```diff
@@ -83,3 +83,31 @@ def do_event(tool, etype, button=1, xdata=0, ydata=0, key=None, step=1):
     event = mock_event(tool.ax, button, xdata, ydata, key, step)
     func = getattr(tool, etype)
     func(event)
+
+
+def click_and_move(tool, start, end, key=None):
+    """
+    Click at *start*, move to end*, and then relese the mouse.
+
+    Parameters
+    ----------
+    tool : matplotlib.widgets.RectangleSelector
```

Comment:
```suggestion
    tool : `matplotlib.widgets.RectangleSelector`
```
But is this really specific to `RectangleSelector`? If so maybe the function name should be `click_and_drag_rectangle` or `click_and_drag_selector`.

Reply:
Thanks for catching, it's not specific to RectangleSelector